src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -310,10 +321,12 @@ std::string tenantFromDb(std::string& database)
 {
   std::string r;
   std::string prefix  = dbPrefix + "-";
+
   if (strncmp(prefix.c_str(), database.c_str(), strlen(prefix.c_str())) == 0)
   {
     char tenant[MAX_SERVICE_NAME_LEN];
```

Comment:
Now that I see this MAX_SERVICE_NAME_LEN is remember that it was too long (1024 bytes). Probably we should make it shorter, to reduce "stress" in the stack space.